lib/collection/src/shards/resharding/driver.rs

```diff
@@ -625,18 +625,19 @@ async fn stage_propagate_deletes(
     };
 
     while let Some(source_shard_id) = block_in_place(|| state.read().shards_to_delete().next()) {
-        let mut offset = None;
+        let mut offset = Some(0.into());
```

Comment:
I think having `None` is better, because we might also have UUIDs. UUIDs should be ordered after integers, but still.

I'll change it to a loop with an explicit break check at the end if you don't mind.

Reply:
Ok.